test/e2e/common/node/container_probe.go

```diff
@@ -71,7 +71,9 @@ var _ = SIGDescribe("Probing container", func() {
 		framework.ExpectNoError(err)
 		isReady, err := testutils.PodRunningReady(p)
 		framework.ExpectNoError(err)
-		framework.ExpectEqual(isReady, true, "pod should be ready")
+		if !isReady {
+			framework.Fail("pod should be ready")
```

Comment:
A nice improvement here would be to include the Pod namespace/name in the output to be able to grep logs.